test/test-application/src/index.ts

```diff
@@ -15,13 +15,17 @@
  */
 
 'use strict';
-import {EchoClient} from 'showcase-echo-client';
-import {ShowcaseServer} from 'showcase-server';
+import { EchoClient, SequenceServiceClient,protos } from 'showcase-echo-client';
+
+
 
 import * as assert from 'assert';
-import {promises as fsp} from 'fs';
+import { promises as fsp } from 'fs';
 import * as path from 'path';
-import {protobuf, grpc, GoogleError, GoogleAuth} from 'google-gax';
+import { protobuf, grpc, GoogleError, GoogleAuth} from 'google-gax';
+import { google } from 'google-gax/build/protos/operations';
```

Comment:
I believe `protos` must be exported from `google-gax` so we could _probably_ do `import {protos} from 'google-gax'; const google = protos.google;` but we can also leave it like this if you prefer.

Reply:
I just got rid of line 26 and things were fine - I think this is leftover from something we tried earlier